docker/lib/dependabot/docker/file_fetcher.rb

```diff
@@ -7,26 +7,44 @@ module Dependabot
   module Docker
     class FileFetcher < Dependabot::FileFetchers::Base
       def self.required_files_in?(filenames)
-        filenames.any? { |f| f.match?(/dockerfile/i) }
+        filenames.any? { |f| f.match?(/dockerfile/i) } or
+          filenames.any? { |f| f.match?(/^[^\.]+\.ya?ml$/i) }
       end
 
       def self.required_files_message
-        "Repo must contain a Dockerfile."
+        "Repo must contain a Dockerfile or Kubernetes YAML files."
       end
 
       private
 
+      def kubernetes_enabled?
```

Comment:
I guess since we only fetch manifests when this is true, we don't need to add an option further down the chain in the FileUpdater etc, as they will simply not be considered?

Reply:
Yes, I believe that is correct, since we won't fetch the manifests, they won't get parsed etc.